pytest_fixtures/core/sat_cap_factory.py

```diff
@@ -37,13 +38,27 @@ def _target_satellite_host(request, satellite_factory):
         yield
 
 
+@lru_cache
+def capsule_cdn_register(hostname=None):
+    cap = Capsule(hostname=hostname)
+    cap.register_to_cdn()
```

Comment:
since it isn't being returned, might as well do it in one line
```suggestion
    Capsule(hostname=hostname).register_to_cdn()
```